Source/spells.h

```diff
@@ -16,6 +16,7 @@ enum class SpellCheckResult : uint8_t {
 	Fail_Busy,
 };
 
+bool IsWallSpell(const spell_id spl);

```

Comment:
nit: There is no need to specify `const` here. It is passed by value (copied) and what the function does with the copy is its own business.

Reply:
```suggestion
bool IsWallSpell(spell_id spl);
```

Reply:
right : D damn I wanted to use const for the first time